django-os2webscanner/os2webscanner/templates/os2webscanner/rules.html

```diff
@@ -9,9 +9,14 @@
   {% include 'os2webscanner/rules_above_content.html' with active='adhoc' %}
   <div class="tab-content">
     <div role="tabpanel" class="tab-pane active" id="adhoc">
-      <p class="h2" style="display:inline-block;">Ad hoc-regler</p>
+      <p class="h2" style="display:inline-block;">Regulær udtryk-regler</p>
```

Comment:
IMO “Regler” is implicit given that we're on the rules page. Also, grammar:
```suggestion
      <p class="h2" style="display:inline-block;">Regulære udtryk</p>
```

Reply:
Ad hoc regler er Mortens udtryk. Så det skal det hedde!
Den anden vil jeg godt acceptere :)